src/FluentValidation.DependencyInjectionExtensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace FluentValidation
+{

```

Comment:
Please can all braces be moved to the end of the preceding line for consistency with the rest of the project.

Reply:
I think this is resolved

Reply:
Braces are still on new lines - please can they be moved to end of line? so `namespace FluentValidation { `